ts/src/coinmetro.ts

```diff
@@ -0,0 +1,1922 @@
+                '4h': '14400000',
+                '1d': '86400000',
+            },
+            'urls': {
+                'logo': '', // todo
+                'api': {
+                    'public': 'https://api.coinmetro.com',
+                    'private': 'https://api.coinmetro.com',
+                },
+                'test': {
+                    'public': 'https://api.coinmetro.com',
+                    'private': 'https://api.coinmetro.com/open',
+                },
+                'www': 'https://coinmetro.com/',
+                'doc': [
+                    'https://documenter.getpostman.com/view/3653795/SVfWN6KS',
+                ],
+                'fees': 'https://help.coinmetro.com/hc/en-gb/articles/6844007317789-What-are-the-fees-on-Coinmetro-',
+            },
+            'api': {
+                'public': {
+                    'get': {
+                        'demo/temp': 1,
+                        'exchange/candles/{pair}/{timeframe}/{from}/{to}': 1,
+                        'exchange/prices': 1,
+                        'exchange/ticks/{pair}/{from}': 1,
+                        'assets': 1,
+                        'markets': 1,
+                        'exchange/book/{pair}': 1,
+                        'exchange/bookUpdates/{pair}/{from}': 1, // not unified
+                    },
+                },
+                'private': {
+                    'get': {
+                        'users/balances': 1,
+                        'users/wallets/history/{since}': 1,
+                        'exchange/orders/status/{orderID}': 1,
+                        'exchange/orders/active': 1,
+                        'exchange/orders/history/{since}': 1,
+                        'exchange/fills/{since}': 1,
+                        'exchange/margin': 1, // not unified
+                    },
+                    'post': {
+                        'jwt': 1, // not unified
+                        'jwtDevice': 1, // not unified
+                        'devices': 1, // not unified
+                        'jwt-read-only': 1, // not unified
+                        'exchange/orders/create': 1,
+                        'exchange/orders/modify/{orderID}': 1, // not unified
+                        'exchange/swap': 1, // not unified
+                        'exchange/swap/confirm/{swapId}': 1, // not unified
+                        'exchange/orders/close/{orderID}': 1,
+                        'exchange/orders/hedge': 1, // not unified
+                    },
+                    'put': {
+                        'jwt': 1, // not unified
+                        'exchange/orders/cancel/{orderID}': 1,
+                        'users/margin/collateral': 1,
+                        'users/margin/primary/{currency}': 1, // not unified
+                    },
+                },
+            },
+            'requiredCredentials': {
+                'apiKey': false,
+                'secret': false,
+                'uid': true,
```

Comment:
I think we need to add `token:true` here

Reply:
Yeah, I think so too )